governance.md

```diff
@@ -0,0 +1,201 @@
+# Governance
+
+This document describes the rules and governance of the project. It is a slightly modified version of the [Prometheus Governance](https://prometheus.io/governance/#governance-changes).
+
+It is meant to be followed by all the developers of the Prometheus Operator project and the Prometheus Operator community. Common terminology used in this governance document are listed below:
+
+* **Maintainers Team**: A core Prometheus Operator team that have owner access to http://github.com/prometheus-operator organization and all projects within it. Current list is available [here][maintainers-doc].
+
+* **Triage Team**: Contributors who does not belong to Maintainer's team, but has `Triage` GitHub role on [Prometheus Operator](https://github.com/prometheus-operator) repository allowing to change GitHub issues and PRs statuses and labels.
+They are listed [here](https://github.com/prometheus-operator/prometheus-operator/blob/master/MAINTAINERS.md#triage).
+
+* **The Prometheus Operator project**: The sum of all activities performed under the [prometheus-operator organization on GitHub][gh], concerning one or more repositories or the community.
+
+Both Triage and Maintainers have are part of [`prometheus-operator-team@googlegroups.com`][team] email list.
+
+## Values
+
+The Prometheus Operator developers and community are expected to follow the values defined in the [CNCF charter][charter], including the [CNCF Code of Conduct][coc].
```

Comment:
Nothing against the CNCF part, just curious as we are not part of that org, are there any better ones out there we could follow?

Reply:
leftover from prometheus/thanos I guess? The CoC link is correct => https://github.com/prometheus-operator/prometheus-operator/blob/master/code-of-conduct.md